packages/grid/_modules_/grid/constants/localeTextConstants.ts

```diff
@@ -41,6 +41,7 @@ export const GRID_DEFAULT_LOCALE_TEXT: GridLocaleText = {
   // Filter panel text
   filterPanelAddFilter: 'Add filter',
   filterPanelDeleteIconLabel: 'Delete',
+  filterPanelLinkOperator: '',
```

Comment:
We need to add a label. Keeping it empty violates https://www.w3.org/TR/using-aria/#fifthrule

The correct name would be "logic operator" but it's too long. A benchmark might be necessary. For me, "operator" is perfect and "operators" (referring to those like "contains") should be changed to  "rule" or "condition". But if we change then we lose sync with the translation constant. Anyway, the API uses some confusing names for the filter model: https://github.com/mui/mui-x/pull/1722#discussion_r640583455

Reply:
"logic operator" seems good. From what I see in other tables, they nether display the label (And/Or) is clear enough, except notion which adds an helper: "All filters much match"

![image](https://user-images.githubusercontent.com/45398769/154023239-0b50064f-9ce7-4c0f-b3c6-bbe44d2c8d9c.png)

We can add it to the aria-label of the input, and not show it on top of the input

Reply:
Removing the label may not be ideal, there's no demo without it. We could experiment with the [ToggleButton](https://mui.com/components/toggle-button/). It doesn't require a visible label. There's also a demo using a [Menu](https://mui.com/components/menus/#selected-menu).

Reply:
I don't think ToggleButton is a good option. We already have a lot of information on the page. Showing both "And" and "Or" options will add confusion.
About the `Menu` demo, I don't see the difference. 

The demo is using a combination of `aria-label` and `aria-labelledby` to compensate the missing label, plus some `role: 'listbox'`, `aria-haspopup="listbox"`, ... to respect a11y.

The current solution add `aria-label` to compensate the missing label, and we use meaning full HTML (`select` and `option`)  to let the screen reader understand the structure.

If your point is that [selects accessibility](https://mui.com/components/selects/#accessibility) does not contain demo without label, I think the topic of this section is mostly how to add a label on Select component

Reply:
It may look a bit weird if the user changes the variant to `filled` but the current approach fixes the accessibility.

<img width="500" src="https://user-images.githubusercontent.com/42154031/154483230-b312b866-bbe7-4033-9db8-3d34e999b624.png" alt="image" style="max-width: 100%;">

Reply:
In such a case it would be nice to have a way to add a label, but I do not see how to make that easy to find for developers. 

What do you think about wanting to see if some users require to display the label?